pytest_mh/utils/firewall.py

```diff
@@ -244,12 +245,17 @@ def __init__(self, host: MultihostHost) -> None:
         not remove "accept" rule but the "drop" rule takes precedence.
         """
 
+        self._policy: str = f"test-policy-{randrange(99999)}"
+        self.add_policy(self._policy, ingress="HOST", egress="ANY")
+        self.host.ssh.exec(["firewall-cmd", "--reload"], log_level=SSHLog.Error)
```

Comment:
Please, move this to setup().

Reply:
I should have guessed that there was a `setup()` function if there was a `teardown()` one.
Done.